src/game/PowerManager.cxx

```diff
@@ -61,12 +39,14 @@ void PowerManager::update()
       addPowerNodeToGrid(nodeToAdd, m_powerGrids);
     }
     m_nodesToAdd.clear();
```

Comment:
This may lead to some concurrency prblems when the user add a tileID between line 40 and line 41. Of course, this is highly unlikely.

Reply:
Are you sure this could be an issue? This is all single threaded and shouldn't be possible imho.